custom-words.txt

```diff
@@ -2039,6 +2039,7 @@ ziplist
 Zoho
 zoneinfo
 zset
+devops
```

Comment:
can you add it in Alphabetical order? despite other people actually did not follow it.
if possible, can we move the rest four words to the correct position? Thanks!